packages/filebrowser-extension/src/index.ts

```diff
@@ -848,3 +827,59 @@ function createLauncher(
       return launcher;
     });
 }
+
+/**
```

Comment:
I have a minor preference for consolidating into a `Private` namespace, just for consistency with the rest of the project. But no strong opinions, since they are not exported either way...